packages/api/db/cargo.testing.sql

```diff
@@ -50,3 +50,31 @@ INSERT INTO public."user" (magic_link_id, github_id, name, email, public_address
 
 INSERT INTO public.auth_key (name, secret, user_id) VALUES ('main', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDY1MDA3QTczOWFiN0FDNWM1MzcxNjEyNDliODEyNTBFNDllMjg1M0MiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTYzOTc1NDczNjYzOCwibmFtZSI6Im1haW4ifQ.wKwJIRXXHsgwVp8mOQp6r3_F4Lz5lnoAkgVP8wqwA_Y', 1);
 
+
+-- Count is the most accurate representation of data
+-- Creating a materialized views gives us cacheing as well as accuracy for percent difference
+
+CREATE MATERIALIZED VIEW upload_stats AS
+	SELECT
+    (
+        SELECT
+        CAST("count"(*) AS float )
```

Comment:
Why cast as float? - it will never have a fractional component.

Reply:
Previously i was doing a calculation in SQL that resulted in a decimal through division, this is no longer needed so I can remove!